src/main/java/seedu/address/storage/JsonAdaptedDriver.java

```diff
@@ -67,7 +76,18 @@ public Driver toModelType() throws IllegalValueException {
         }
         final int modeDriverId = Integer.parseInt(driverId);
 
-        return new Driver(modeDriverId, modelName, modelPhone, modelEmail, modelAddress, modelTags);
+        Driver driver = new Driver(modeDriverId, modelName, modelPhone, modelEmail, modelAddress, modelTags);
+
+        if (rating == null || totalNoOfReviews == null) {
+            return driver;
+        }
+
+        final int modeDriverRating = Integer.parseInt(rating);
+        final int modeDriverNoOfReviews = Integer.parseInt(totalNoOfReviews);
```

Comment:
Integer parseInt will throw NumberFormatException error, so have to check its validity first. 
otherwise, can use a try catch

Reply:
done